tubemq-core/src/main/java/org/apache/tubemq/corebase/config/constants/BrokerCfgConst.java

```diff
@@ -1,3 +1,20 @@
+/**
```

Comment:
The constant definitions and token definitions of the server module should be summarized in the server module and do not need to be known by the client